bwrap.xml

```diff
@@ -469,6 +469,14 @@
         (see CVE-2017-5226).
       </para></listitem>
     </varlistentry>
+    <varlistentry>
+      <term><option>--foreground</option></term>
```

Comment:
I wonder whether this should have a name more like `--terminal-foreground` or `--tty-foreground`, to indicate that it isn't to do with daemonization or other things that might be referred to as "foreground" or "background"?